test/functional/JIT_Test/src/jit/test/recognizedMethod/TestJavaLangMath.java

```diff
@@ -55,4 +59,226 @@ public void test_java_lang_Math_sqrt() {
         AssertJUnit.assertEquals(Double.POSITIVE_INFINITY, Math.sqrt(Double.POSITIVE_INFINITY));
         AssertJUnit.assertTrue(Double.isNaN(Math.sqrt(Double.NaN)));
     }
+
+    /**
+    * Tests all execution paths defined by the {@link Math.max} methods, for float and double data types.
+    */
+    @Test(groups = {"level.sanity"}, invocationCount=2)
+    public void test_java_lang_Math_max() {
```

Comment:
Not sure if this is what you meant, but I already have a Utils class with the constants and wrappers. Gonna change this file to use `import static` on that file so I can make it more readable

Reply:
What I meant is that, current test you have, needs wrapper to ensure that fmax node is not simplified to fconst (Which for some of the hardcoded changes it will be). Can we use `@DataProvide` from testNG? There must be many examples in https://github.com/ibmruntimes/openj9-openjdk-jdk21 as well as couple in openj9 tests as well). 

That way it will be easy to maintain and read.